src/Devices/PHProbe.cpp

```diff
@@ -28,16 +29,83 @@ PHProbe::PHProbe() {
   Serial1.print(F("C,1\r"));    // Reset pH stamp to continuous measurement: once per second
 }
 
+void PHProbe::clearCalibration() {
+  Serial1.print("Cal,clear\r");  // send that string to the Atlas Scientific product
+}
+
+String PHProbe::getSlope() {
```

Comment:
Do you have some pending changes to this function? I thought we set up a `sendReadSlope()` function that did not block. This function blocks when it calls serialEvent1().